src/main/java/frc/robot/commands/drivetrain/ManualDrive.java

```diff
@@ -36,7 +37,7 @@ public void initialize() {
   public void execute() {
     this.drivetrain.setDrivetrainOutput(
         this.controller.getDrivetrainOutput(),
-        true);
+        !this.controller.getRawButton(ControllerConstants.intake));
```

Comment:
This is a great opportunity to declare a variable simply for documentation. You could say something like

```
boolean intakeDisabled =  !this.controller.getRawButton(ControllerConstants.intake);

this.drivetrain.setDrivetrainOutput(this.controller.getDrivetrainOutput(), intakeDisabled);
```

It doesn't fundamentally change what you've done but makes it a lot easier for the next person working on it to understand what exactly is happening.